omnichannel-server/src/feature/test/MessageSourceDigest.test.ts

```diff
@@ -25,9 +25,17 @@ describe("MessageSourceDigest", () => {
       currentTime: new Date(currentTime),
       from,
     };
-    
+
     expect(messageSourceDigest.parseRequestBody(messageBirdPayload)).toEqual(
       payload
     );
   });
+
+  test("Should return undefined", () => {
+    const unknowPayload: any = {
```

Comment:
En realidad todos los payload son `unknown` (desconocidos), pero pueden ser inválidos:

```suggestion
    const invalidPayload: any = {
```